graphql/typeDefs.ts

```diff
@@ -168,10 +168,12 @@ export default gql`
   }
 
   type Star {
-    id: String!
-    studentId: Int
-    mentorId: Int
-    lessonId: Int
+    id: Int!
+    studentId: Int!
+    mentorId: Int!
+    lessonId: Int!
```

Comment:
Are these ID values used on the app? Seems kinda unnecessary to me to have ids together with the nested value.

Reply:
just lessonId . removing the other two.